fossology/fossology
# feat(decisions): add new decision type non-functional

### Changes

add new decision "non functional" to decision types

## How to test

* Upload a component.
* Apply Non functional decision from both single file view and tree view.
* Check if the results are attached properly to the files.
* Check unified report for non-functional files and comments.

Closes #1114
